modular_skyrat/modules/cargo/code/goodies.dm

```diff
@@ -87,7 +87,7 @@
 /datum/supply_pack/goody/nri_gear
 	name = "NRI Voentorg Gear"
 	desc = "Contains a backpack full of military-grade NRI Defense Collegium standard equipment previously used by the most of their military."
-	cost = PAYCHECK_CREW * 40
+	cost = PAYCHECK_CREW * 30
```

Comment:
Why was this lowered in price so significantly?

Reply:
You can get a single SWAT crate of better equipment for *two* people for a single NRI gear crate's price. Which is kinda an issue.